otherlibs/systhreads/st_stubs.c

```diff
@@ -29,7 +29,24 @@
 #define DLL_EXPORT
 #endif
 
```

Comment:
```suggestion
#include "caml/config.h"
```

for the configure-defined macros